src/rgw/rgw_sal_motr.cc

```diff
@@ -1452,16 +1452,7 @@ int MotrObject::MotrReadOp::prepare(optional_yield y, const DoutPrefixProvider*
   if(source->get_obj_size() == 0)
```

Comment:
Remove lines 1451, 1452 & 1453  .. as these are not required

Reply:
done. please check